cves/CVE-2017-9805.yml

```diff
@@ -119,9 +123,13 @@ unit_tested:
     For the "fix" answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
     Must be just "true" or "false".
-  answer:
-  code:
-  fix:
+  answer: |
+    There were no unit tests involved in this vulnerability. There are unit
```

Comment:
For my vulnerability I used the same logic to argue that unit tests were involved. There were unit tests for the module, they just lacked a security test. This is something you could mention in the mistakes section as well.